Google.Cloud.Spanner.NHibernate.Tests/NHibernateMockServerFixture.cs

```diff
@@ -46,6 +46,8 @@ public NHibernateMockServerFixture()
             var mapper = new ModelMapper();
             mapper.AddMapping<SingerMapping>();
             mapper.AddMapping<AlbumMapping>();
+            mapper.AddMapping<SingerWithVersionMapping>();
```

Comment:
If you add these new mappings after the mapping for `TableWithAllColumnTypes`, then you won't have to update the aliases that are generated for `TableWithAllColumnTypes`. See also my comment below.